### ISSUE TITLE
Suggestions, take a few hundred online e-books and get all words from it too

### ISSUE THREAD

**Issue by MyTinyBrain** [dwyl/english-words#44]

**Comment by nelsonic** [comment]
@MyTinyBrain that would be _interesting_ but is kinda "beyond the scope" of this repository.

**Comment by JonathanRys-ATC** [comment]
Yeah, there are some important words missing here, like "tree" and "trie", which was slightly embarrassing when I was demoing my trie algorithm using this data and asked people to pick a word with "tree/trie" being the obvious choice.  Although, sucking in data from ebooks split on spaces and added to a Set would yield a decent result, you might find non-English words too.  "A Clockwork Orange", "The Jabberwocky", or Dr. Seuss, for instance, would yield nonsense words that would be hard to filter out.  Scraping dictionary.com might be a better approach.

**Comment by scottchiefbaker** [comment]
I wrote a Perl script to do [exactly](https://www.perturb.org/code/extract_words.pl) what you're talking about for another project. It extracts anything that LOOKS like a word, so there is no validation that it isn't a proper name or an [onomatopoeia](https://www.merriam-webster.com/dictionary/onomatopoeia).
